engine/engine/src/engine.cpp

```diff
@@ -950,15 +958,6 @@ namespace dmEngine
 
         SetUpdateFrequency(engine, engine->m_UpdateFrequency);
 
-        bool setting_vsync = dmConfigFile::GetInt(engine->m_Config, "display.vsync", true); // Deprecated
-
-        uint32_t opengl_swap_interval = dmConfigFile::GetInt(engine->m_Config, "display.swap_interval", 1);
-        if (!setting_vsync)
-        {
-            opengl_swap_interval = 0;
-        }
-        SetSwapInterval(engine, opengl_swap_interval);
```

Comment:
The setting is passed in to the context creation params instead of calling the function directly. We need to know this when creating the swap chain in vulkan